OIPA/api/region/views.py

```diff
@@ -8,17 +8,60 @@
 
 
 class RegionList(ListAPIView):
+    """
+    Returns a list of IATI Regions stored in OIPA.
+
+    ## Result details
+
+    Each result item contains short information about region
+    including URI to region details.
+
```

Comment:
request parameters: 
`fields` request parameter is available here.
aggregation is possible. `/api/regions?fields[aggregations]=total_budget`